litmus-portal/frontend/src/pages/MonitoringDashboardPage/styles.ts

```diff
@@ -10,15 +10,16 @@ const useStyles = makeStyles((theme) => ({
   },
 
   root: {
-    marginTop: theme.spacing(5),
-    marginLeft: theme.spacing(6),
-    marginBottom: theme.spacing(8),
+    marginTop: theme.spacing(-1),
```

Comment:
Why should the root element itself be given a negative margin? Please check

Reply:
please check on this

Reply:
The scaffolding has default padding/margin due to which we require this negative margin. cc: @vanshBhatia-A4k9

Reply:
the scaffolding has those to bring uniformity...that is not to be changed.

Reply:
To match it as per design then we need this margin.

Reply:
I don't think so...all other pages are not having any negative margins. Show me how much difference it is making...if it is really not according to the design we'll change the scaffold...but one page can't be different from the others.

Reply:
This change is pending

Reply:
should i remove the change but then it wont match the design, there is some problem with the scaffold PTAL here, it is pushing the content down due to extra padding. As you may check, to make it as per the design we had to provide this negative margin on the monitoring page. on all other pages content is being pushed down automatically due to the scaffolding.


<img width="666" alt="Screenshot 2021-04-06 at 3 14 41 PM" src="https://user-images.githubusercontent.com/50652677/113692441-2f4d2d00-96eb-11eb-8b3e-454843a7ce6b.png">
<img width="608" alt="Screenshot 2021-04-06 at 3 15 15 PM" src="https://user-images.githubusercontent.com/50652677/113692459-32481d80-96eb-11eb-85d0-4a4d66b2a786.png">
<img width="832" alt="Screenshot 2021-04-06 at 12 31 31 PM" src="https://user-images.githubusercontent.com/50652677/113692463-32e0b400-96eb-11eb-9adb-577b8c09f938.png">
<img width="690" alt="Screenshot 2021-04-06 at 12 31 58 PM" src="https://user-images.githubusercontent.com/50652677/113692468-3411e100-96eb-11eb-9d4a-5eb1b6eb47c6.png">
<img width="690" alt="Screenshot 2021-04-06 at 12 32 12 PM" src="https://user-images.githubusercontent.com/50652677/113692472-34aa7780-96eb-11eb-8dab-4644d8f75a5c.png">
<img width="891" alt="Screenshot 2021-04-06 at 12 32 51 PM" src="https://user-images.githubusercontent.com/50652677/113692476-34aa7780-96eb-11eb-9388-f46efb789a0e.png">

Reply:
Scaffold starts the page where it needs to start now to maintain the consistency across all pages, simply removing all the margins from the left and top will place all the places where they need to be.

Reply:
@vanshBhatia-A4k9 but its not working for the monitoring page please check the scaffold again for now i am removing this negative margin fix. cc: @arkajyotiMukherjee